logbook-logstash/src/test/java/org/zalando/logbook/logstash/LogbackLogstashSinkTest.java

```diff
@@ -200,4 +200,101 @@ void shouldLogCorrectJsonForEmptyBody() throws IOException {
         }
 
     }
+
+    @Test
+    void shouldUsePassedBaseField() throws IOException {
```

Comment:
Is this a full copy of the original test? I'd either make this test way smaller so that it's laser-focused on what it tries to test (just the name of the field) or as an alternative, extract the common parts of both tests into a separate, private method and call that twice with a different parameter:


```java
@Test
void shouldUsePassedBaseField() throws IOException {
    logsAs("test")
}
```

Reply:
I like the private method approach, will go for it